src/PresentationalComponents/Notifications/utils.js

```diff
@@ -51,6 +51,7 @@ export const prepareFields = (notifPref, emailPref, emailConfig) =>
               component: 'tabGroup',
               fields: [
                 ...fields,
+                unsubscribeAllInactive == true ?
```

Comment:
we may want to remove these checkboxes completely and use buttons instead per mockups